### PR TITLE
{bio}[dummy/] picard v2.18.27

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2045.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b10efadb9f2fa336b40837ba2ca2bcdc for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3146.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/285bda560803d7f587eb8b06f1d1c448 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2401.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1e9f65497f38d6af8559a6973eee0082 for a full test report.

File: easybuild/easyconfigs/p/picard/picard-2.18.27-Java-1.8.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+source_urls = ['https://github.com/broadinstitute/picard/releases/download/%(version)s']
+sources = [{
+    'filename': '%(name)s-%(version)s.jar',
+    'download_filename': '%(name)s.jar',
+}]
+checksums = ['6cfba52d348969bf7b17adc26d1e8e669f741631253003e4a10dd8322eac89dd']
+
+postinstallcmds = ["mv %(installdir)s/%(name)s-%(version)s.jar %(installdir)s/%(name)s.jar"]
+
+dependencies = [('Java', '1.8', '', True)]
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/picard/picard-2.18.27-Java-1.8.eb]: Do you really need  '', True when it is already in dummy? Or is it just to be explicit?

Comment (boegel) [reply easybuild/easyconfigs/p/picard/picard-2.18.27-Java-1.8.eb]: Not needed, you're right, will fix.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar04.gastly.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 2.7.5
See https://gist.github.com/dfcf148360c27cdb00f7e31ed731fa89 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/8e297353e2b0cb343afb33e5f87ded9c for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Pull request merged**